comms/Autoupdate.cpp

```diff
@@ -146,3 +146,33 @@ IAutoUpdate * CreateIAutoUpdate()
 {
     return new CAutoUpdate();
 }
+
+COMMS_API bool DNSLookup(std::_tstring & url) {
+    size_t found = url.find_first_of(_T(":"));
```

Comment:
Might be clearer/safer if this said "://" rather than ":" (assuming I am understanding the purpose correctly)